renderer.js

```diff
@@ -121,11 +133,15 @@ function enterFullscreen () {
 
 function exitFullscreen () {
   if (isFullscreen()) return;
-  fixUpCanvas();
-  resetCamera(camera);
+  handleResize();
   safeToRenderStereo = false;
 };
 
+function handleResize() {
+  fixUpCanvas();
```

Comment:
for some reason, I still see my commits in here:

https://github.com/nickdesaulniers/joshVR/pull/1/commits